nexus/reconfigurator/planning/src/blueprint_builder/zones.rs

```diff
@@ -346,6 +346,11 @@ mod tests {
                 .expect("new zone ID should be present");
         }
 
+        // TODO: FIXME: AJS - Delete use of the control sled? - We don't
```

Comment:
Do you want to address this on this PR?

Reply:
I removed this section of code, as it's unused.